config/deploy/staging.rb

```diff
@@ -3,4 +3,8 @@
 ask :branch_name, proc { `git rev-parse --abbrev-ref HEAD`.chomp }
 set :branch, proc { fetch(:branch_name); `git rev-parse #{fetch(:branch_name)}`.chomp }
 
-set :default_env, { welcome_mat: true, tcf_demo: true, path: "/opt/node/current/bin:/opt/python/current/bin:$PATH" }
+set :default_env, {
+  welcome_mat: true,
+  tcf_demo: true,
+  id_client_key: 'X5rYnqHGVBpqPTfHKYQyxG6PYfVj0l8nostkQzTC',
```

Comment:
In staging, we need to ensure that we use an application that redirects to "https://alpha.prx.org/...". The default value (set in flags.release.json) redirects to https://beta...